tests/integration/support/utils.go

```diff
@@ -0,0 +1,79 @@
+package support
+
+import (
+	"encoding/json"
+	"regexp"
+	"strings"
+
+	"github.com/optimizely/go-sdk/pkg/event"
+	"github.com/optimizely/go-sdk/tests/integration/models"
+	"gopkg.in/yaml.v3"
+)
+
+// Reset clears all data before each scenario
+func (c *ScenarioCtx) Reset() {
+	c.apiOptions = models.APIOptions{}
+	c.apiResponse = models.APIResponse{}
+	c.clientWrapper = ClientWrapper{}
+}
+
+func (c *ScenarioCtx) getDispatchedEventsFromYaml(s string) ([]event.Batch, error) {
```

Comment:
These shouldn't be part of the `ScenarioCtx` struct. Instead pass in the project config reference in the args